CMakeLists.txt

```diff
@@ -79,7 +79,11 @@ if (DEFINED PYTHON_INCLUDE_DIR AND DEFINED PYTHON_LIBRARIES AND DEFINED Python_E
     SET(PYTHONLIBS_FOUND TRUE)
     # Use the Python module to find the python lib.
     if(BUILD_WITH_PYTHON_3)
-        find_package(Python 3.7 EXACT REQUIRED COMPONENTS Interpreter)
+        # We support two versions of python 3 (3.7/3.9).
+        find_package(Python 3.9 EXACT QUIET COMPONENTS Interpreter)
+        if (NOT Python_FOUND)
+            find_package(Python 3.7 EXACT REQUIRED COMPONENTS Interpreter)
+        endif()
```

Comment:
First search for python 3.9 with quiet option so if not found, no message or error will be output. Then if python was not found we search again for 3.7 with required (will error if not found). This case is for our internal build system where we send in the exact python to use via the variables (line 77).